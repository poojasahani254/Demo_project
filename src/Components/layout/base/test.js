import React from 'react'
import { shallow } from 'enzyme'
import Renderer from 'react-test-renderer'
import { BaseLayoutComponent } from '.'

const setup = propOverrides => {
  const props = {
    ...propOverrides
  }

  const base = (
    <BaseLayoutComponent {...props}>
      this is Layout component
    </BaseLayoutComponent>
  )
  const component = shallow(base)
  const instance = component.instance()
  const tree = Renderer.create(base).toJSON()
  return {
    tree,
    props,
    component,
    instance
  }
}

describe('Base Layout Component ', () => {
  it('Snapshot', () => {
    const { tree } = setup()
    expect(tree).toMatchSnapshot()
  })
  it('should render correctly', () => {
    const { component } = setup()
    expect(component.find('.wrap')).toHaveLength(1)
  })
})
