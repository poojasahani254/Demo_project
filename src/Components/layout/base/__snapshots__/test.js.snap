// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Base Layout Component  Snapshot 1`] = `
<section
  className="wrap ant-layout"
>
  this is Layout component
</section>
`;
